modules/microshift-oc-mirror-config-cluster.adoc

```diff
@@ -0,0 +1,108 @@
+//Module included in the following assemblies:
+//
+//* microshift_running_apps/microshift-operators-olm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-oc-mirror-config-cluster_{context}"]
+= Configuring your {microshift-short} cluster to use Operators selected with oc mirror
+
+After you mirror your image set to the mirror registry, you must apply the generated `CatalogSource` custom resource (CR) into the cluster. The `CatalogSource` CR is used by Operator Lifecycle Manager (OLM) to retrieve information about the available Operators in the mirror registry. You can then create a Subscription custom CR and apply that.
+
+.Prerequisites
+
+* You mirrored the image set to your registry mirror.
+* You added image reference information to the CRI-O container runtime configuration.
+
+.Procedure
+
+. Change the `openshift-marketplace` namespace label `pod-security.kubernetes.io/enforce` from `restricted` to `baseline` in the `CatalogSource` CR:
```

Comment:
If the documentation lands the same time as the https://github.com/openshift/microshift/pull/3076 then I'm not sure we need to mention anything in this regard because that CatalogSource will just work.